tests/conftest.py

```diff
@@ -35,3 +50,30 @@ def df_spark(spark):
     df = spark.read.csv("./tests/datasets/fake_1000_from_splink_demos.csv", header=True)
     df.persist()
     yield df
+
+
+# does this even need to be a fixture?? :thinking_face:
+@pytest.fixture
+def duckdb_helper():
+    return DuckDBTestHelper()
+
+
+@pytest.fixture
+def spark_helper(spark):
+    return SparkTestHelper(spark)
+
+
+@pytest.fixture
+def sqlite_helper():
+    return SQLiteTestHelper()
+
+
+# workaround as you can't pass fixtures as param arguments in base pytest
+# see e.g. https://stackoverflow.com/a/42400786/11811947
+@pytest.fixture
+def test_helpers(duckdb_helper, spark_helper, sqlite_helper):
+    return {
+        "duckdb": duckdb_helper,
+        "spark": spark_helper,
+        "sqlite": sqlite_helper,
+    }
```

Comment:
Unless we plan on using `duckdb_helper`, `spark_helper`, etc as their own independent fixtures, I'd be tempted to rm them and yield:
```
    return {
        "duckdb": DuckDBTestHelper(),
        "spark": SparkTestHelper(spark),
        "sqlite": SQLiteTestHelper(),
    }
```

Reply:
This is primarily to mitigate confusion

Reply:
Yep I think you are right - think this structure is probably a hangover from trying to get the parameterisation to select the fixtures directly

Reply:
Have made this change